packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -523,6 +524,128 @@ export const trackEventsMapper = {
     wishlistId: data.properties?.wishlistId,
     lineItems: getProductLineItems(data),
   }),
+  [eventTypes.PRODUCT_CLICKED]: data => ({
+    tid: 2926,
+    actionArea: data.properties?.from,
+    lineItems: getProductLineItems(data),
+  }),
+  [eventTypes.FILTERS_APPLIED]: data => ({
+    tid: 2921,
+    actionArea: data.properties?.from,
+    filtersApplied: data.properties?.filters
+      ? JSON.stringify(data.properties?.filters)
+      : undefined,
+  }),
+  [eventTypes.FILTERS_CLEARED]: data => ({
+    tid: 2917,
+    actionArea: data.properties?.from,
+    filtersApplied: data.properties?.filters
+      ? JSON.stringify(data.properties?.filters)
+      : undefined,
+  }),
+  [eventTypes.LOGIN]: data => ({
+    tid: 2924,
+    loginType: data.properties?.method,
+  }),
+  [eventTypes.SIGNUP_FORM_COMPLETED]: data => ({
+    tid: 2927,
+    loginType: data.properties?.method,
+  }),
+  [eventTypes.INTERACT_CONTENT]: data => {
+    const properties = data.properties;
+    if (!properties?.contentType && !properties?.interactionType) {
+      logger.warn(
+        `[Omnitracking] - Event ${data.event} requires at lease properties contentType and interactionType
+                        Please fill this information to track event on omnitracking's integration.`,
+      );
+      return;
+    }
+
+    return {
+      tid: 2882,
+      contentType: data.properties?.contentType,
```

Comment:
As you have the `properties` variable available, you could use that instead of using `data.properties`.

Reply:
Same as previous item. O Spotted this on scroll task, then I resolve there.

I'm waiting for rebase to fix this and get branch ready to merge.

Reply:
Alright! Closing it...